<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoRecorder</class>
 <widget class="QMainWindow" name="VideoRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVideoWidget" name="displayvideo" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 255, 161)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>1031</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startCamera">
       <property name="font">
        <font>
         <pointsize>19</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 85);</string>
       </property>
       <property name="text">
        <string>Start Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopCamera">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 85);</string>
       </property>
       <property name="text">
        <string>Stop Camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>120</y>
      <width>357</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="Abdomen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string>Abdomen</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Lower_Extremities">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string> Lower Extremities</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="Head">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>false</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string>Head</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Upper_Extremities">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string> Upper Extremities</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Thorax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string>Thorax</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Spine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string>Spine</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Pelvis">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string>Pelvis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Face">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string>Face</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="General">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(84, 84, 84);
}
</string>
       </property>
       <property name="text">
        <string>General</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stopCamera</tabstop>
  <tabstop>Head</tabstop>
  <tabstop>startCamera</tabstop>
  <tabstop>Abdomen</tabstop>
  <tabstop>Face</tabstop>
  <tabstop>Pelvis</tabstop>
  <tabstop>Thorax</tabstop>
  <tabstop>Spine</tabstop>
  <tabstop>Upper_Extremities</tabstop>
  <tabstop>Lower_Extremities</tabstop>
  <tabstop>General</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
